ares/physics/HaloMassFunction.py

```diff
@@ -366,6 +366,34 @@ def _load_hmf_wdm(self): # pragma: no cover
             self._tab_MAR = 10**(np.diff(log_tmar, axis=0).squeeze() \
                 * (m_X - m_X_l) + log_tmar[0])
 
+    def _get_ngtm_mgtm_from_dndm(self):
+
+        # Generates ngtm and mgtm with Murray's formulas
+        ngtm = []
+        mgtm = []
+        for i in range(len(self.tab_z)):
+            m = self.tab_M[np.logical_not(np.isnan(self.tab_M))]
+            dndm = self.tab_dndm[i][np.logical_not(np.isnan(self.tab_dndm[i]))]
+            dndlnm = m * dndm
+            if m[-1] < m[0] * 10 ** 20 / m[3]:
```

Comment:
```suggestion
            if m[-1] < m[0] * 10 ** 18 / m[3]:
```
Hello! I tried the code but it didn't work. I found that the problem might be this one as the original code from Murray's formula uses "18". After the changing, it works on other HMF now. Thank you!

Reply:
Thanks! Works for me too. Will merge in momentarily.